runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/ExecutableStageDoFnOperator.java

```diff
@@ -339,7 +339,8 @@ private void prepareStateBackend(K key) {
           // Key for state request is shipped already encoded as ByteString,
           // this is mostly a wrapping with ByteBuffer. We still follow the
           // usual key encoding procedure.
-          final ByteBuffer encodedKey = FlinkKeyUtils.encodeKey(key, keyCoder);
+          // final ByteBuffer encodedKey = FlinkKeyUtils.encodeKey(key, keyCoder);
```

Comment:
Can be removed

Reply:
I'm going to leave this here until ByteString is made explicit in `StateRequestHandlers`.